ui/components/app/edit-gas-fee-popover/network-statistics/network-statistics.js

```diff
@@ -38,14 +39,7 @@ const NetworkStatistics = () => {
           </span>
         </div>
         <div className="network-statistics__info__separator" />
-        <div className="network-statistics__info__field network-statistics__info__field--priority-fee">
-          <span className="network-statistics__info__field-data">
-            <PriorityFeeTooltip>0.5 - 22 GWEI</PriorityFeeTooltip>
-          </span>
-          <span className="network-statistics__info__field-label">
-            <I18nValue messageKey="priorityFee" />
-          </span>
-        </div>
+        <LatestPriorityFeeField />
```

Comment:
I opted to replace the `network-statistics__info__field` instead of what was inside the field, which I noticed StatusSlider was doing. If this makes sense, I will come back and fix StatusSlider so it's consistent with this.